src/utils/lib.mli

```diff
@@ -63,6 +63,9 @@ val get : 'a option -> 'a
 (** Return the min between two optional floats. Return None if both floats are None. *)
 val min_option : float option -> float option -> float option
 
+(** Maps a function over an optional value *)
+val map_option: ('a -> 'b) -> 'a option -> 'b option 
+  
```

Comment:
Mention this function is available in version 4.08 and later of the standard library:
https://caml.inria.fr/pub/docs/manual-ocaml/libref/Option.html

Reply:
yup. but we are using 4.07 right? (:

Reply:
I meant please write a comment to future developers.

Reply:
Ah, i misread your comment..